volume/drivers/proxy.go

```diff
@@ -205,48 +211,50 @@ type volumeDriverProxyGetResponse struct {
 	Err    string
 }
 
+// Get retrieves the volume with the requested name
 func (pp *volumeDriverProxy) Get(name string) (volume *proxyVolume, err error) {
 	var (
-		req volumeDriverProxyGetRequest
-		ret volumeDriverProxyGetResponse
+		request  volumeDriverProxyGetRequest
+		response volumeDriverProxyGetResponse
 	)
 
-	req.Name = name
+	request.Name = name
 
-	if err = pp.CallWithOptions("VolumeDriver.Get", req, &ret, plugins.WithRequestTimeout(shortTimeout)); err != nil {
+	if err = pp.CallWithOptions("VolumeDriver.Get", request, &response, plugins.WithRequestTimeout(shortTimeout)); err != nil {
 		return
 	}
 
-	volume = ret.Volume
+	volume = response.Volume
 
-	if ret.Err != "" {
-		err = errors.New(ret.Err)
+	if response.Err != "" {
+		err = errors.New(response.Err)
 	}
 
 	return
 }
 
-type volumeDriverProxyCapabilitiesRequest struct{}
+type volumeDriverProxyCapabilitiesRequest struct {}
```

Comment:
This is not correct.

Reply:
It's probably about the whitespace here (which indeed looks odd)